pkg/datapath/linux/config/config.go

```diff
@@ -264,6 +264,10 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 		cDefinesMap["SERVICE_NO_BACKEND_RESPONSE"] = "1"
 	}
 
+	if option.Config.PolicyDenyResponse == option.PolicyDenyResponseIcmp {
+		cDefinesMap["POLICY_DENY_RESPONSE"] = "1"
+	}
```

Comment:
We are no longer adding new compile time configuration unless there is a very compelling reason not to. We have switched over to using load-time configuration, please see https://docs.cilium.io/en/stable/contributing/development/datapath_config/

To do so you would add a `DECLARE_CONFIG` / `NODE_CONFIG` to the BPF header files. Then use the `CONFIG` macro to access the config values. The new config value on the go side is generated when calling `make -C bpf/`.

To add complexity test coverage you modify https://github.com/cilium/cilium/blob/main/pkg/datapath/loader/verifier_load_test.go instead of adding to the `bpf/complexity-test` files.